packages/KTable/KTable.vue

```diff
@@ -35,7 +36,15 @@
               :name="value.key"
               :row="row"
               :rowKey="rowIndex"
-              :rowValue="row[value.key]">{{ row[value.key] }}</slot>
+              :rowValue="row[value.key]">
+              <a
+                v-if="row[value.key] && row[value.key].href"
+                :href="row[value.key].href">{{ row[value.key].label }}</a>
+              <span v-else-if="row[value.key] && row[value.key].submenu">&#8230;</span>
```

Comment:
what is a sub menu? Why would we use this?

Reply:
Leftover code, will remove